test/e2e/storage/external/external.go

```diff
@@ -0,0 +1,217 @@
+/*
```

Comment:
Can we add a README under .../storage/external?

Reply:
+1, an explanation of how to use this and maybe even a link to an example (when there is one) would be very helpful

Reply:
fyi, @wongma7 and @mathu97 are working on documentation/example

Reply:
That's good, but I'll also add short README here.

Reply:
Done.